packages/liveblocks-core/src/__tests__/_utils.ts

```diff
@@ -80,7 +74,7 @@ export class MockWebSocket implements WebSocket {
     public url: string,
     private onSend: (message: string) => void = () => {}
   ) {
-    this.readyState = this.CLOSED;
+    this.readyState = this.CONNECTING;
```

Comment:
This is our testing helper to mock an actual WebSocket instance, and to allow us to control it during unit testing. I'm planning to rewrite some of it to make that easier to do soon. While I was taking a look here and investigating, I noticed that it would initialize a new WebSocket connection in `CLOSED` state by default (aka when doing `new WebSocket()`). We would typically not notice, because most tests will call `ws.open()` immediately, putting it in `OPEN` state.

This behavior is incorrect however, as new WebSockets in real life will never start in `CLOSED` state, but in `CONNECTING` state instead. This little adjustment will make it behave more closely to a real WebSocket implementation.

(There are many more small issues with this mock that I would like to fix. I just changed this one here and now as it didn't have an effect on our unit tests. It's unrelated to the rest of these refactorings.)